knime_extension/src/nodes/spatialmodels.py

```diff
@@ -16,6 +16,22 @@
 __NODE_ICON_PATH = "icons/icon/SpatialModel/"
 
 
+@knext.parameter_group(label="ID Setting")
+class IDSetting:
+    """
+    The unique ID column. It should always keep the same as the ID column in the spatial weights matrix node.
+    The selected column should contain unique IDs for each observation in the input data.
+
+    """
+
+    Field_col = knext.ColumnParameter(
```

Comment:
Load parameter backward compatible. For an example see here: https://github.com/spatial-data-lab/knime-geospatial-extension/blob/main/knime_extension/src/nodes/spatialtool.py#L601

Reply:
😯 Sorry, I am confused with this example, we added a new parameter, does it still have a backward compatible problem, how to solve it?

Reply:
Just add parameter backward compatible, please check it! @koettert